docs/evaluation/how_to_guides/evaluation/summary.mdx

```diff
@@ -0,0 +1,76 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to run an aggregate evaluation
```

Comment:
Any way to make it clear to users who want to calculate things like f1 score that they should go here from our indexes? I feel like no one ever finds this page